packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -104,15 +104,7 @@ const attributeRouter = (
             req.params.name
           );
 
-          if (attribute) {
-            return res.status(200).json(toApiAttribute(attribute.data)).end();
-          } else {
-            const errorRes = makeApiProblem(
-              attributeNotFound,
-              getAttributesByNameErrorMapper
-            );
-            return res.status(errorRes.status).json(errorRes).end();
-          }
```

Comment:
Refactor. There is no need to throw `attributeNotFound` inside the router. It can be thrown inside `attributeRegistryService`.

P.S. there was also a bug here because `attributeNotFound` should be called, and not passed as a function.